manifests/stable/metallb-operator.clusterserviceversion.yaml

```diff
@@ -273,7 +273,7 @@ metadata:
     categories: Networking
     certified: "false"
     containerImage: quay.io/openshift/origin-metallb-operator:4.13
```

Comment:
Don't we want this to be `quay.io/openshift/origin-metallb-operator:4.14`?

Reply:
I see more differences from the [4.13 bump](https://github.com/openshift/metallb-operator/pull/128/commits/8caf9ac37437b9af40408ebd31eb9ec0ac28409e#diff-39d3553ca219dc0755f36a14e407731b038f0b0a4a821db47f291922a0670c63)